### DISCUSSION TITLE
The ablity to bind backspace to back

### DISCUSSION THREAD

**Discussion by Daisouji** [zen-browser/desktop#3099] [General]
I think this would be a great feature.

**Comment by arcos9000** [comment]
How something that has been a default shortcut in browsers for nearly 30 years isn't even an option is a bit silly.  This guys [article](https://www.kevinthetechguy.ca/p/zen-browser-i-love-you-for-now-but-stop-messing-with-basic-functionality-and-user-habits) hits the nail on the head... Customization should be prioritized over what the developer perceives as improvements to the browsing UX, particularly if those "improvements" are changing decades of convention.  It would be a good rule of thumb for any forks, meant for general consumption, of firefox (or any browser for that matter), to question "is this how mozilla would do it?" and if so, do the opposite or at least give it more consideration. That's why we are all looking for alternatives... right?
